assets/FcmPushAsset.php

```diff
@@ -34,6 +34,15 @@ public static function register($view)
                 'senderId' => $pushDriver->getSenderId(),
             ]);
 
+            Yii::$app->view->registerJsConfig('firebase', [
+                'text' => [
+                    'status.granted' => Yii::t('FcmPushModule.base', 'Granted: Push Notifications are active on this browser.<br>You can disable it in browser settings for this site.'),
+                    'status.denied' => Yii::t('FcmPushModule.base', 'Denied: You have blocked Push Notifications.<br>You can enable it in browser settings for this site.'),
+                    'status.default' => Yii::t('FcmPushModule.base', 'Default: Push Notifications are not yet enabled.<br><button id="enablePushBtn" class="btn btn-primary"><i class="fa fa-unlock"></i> Click here to enable</button>'),
```

Comment:
@ArchBlood The button looks big:

![button](https://github.com/user-attachments/assets/3310a745-53b3-4917-a50a-3ab641b8ea1a)

Maybe we should use style `btn-sm` or `btn-xs`:

![btn-sm](https://github.com/user-attachments/assets/eb03404a-f398-4ac8-9102-dc12f896d133)
![btn-xs](https://github.com/user-attachments/assets/2cfab755-4b19-4f82-a0cf-c93ab82dbc65)

---

Also I think we may have issue on translation like [here](https://github.com/humhub/translation/issues/40#issuecomment-781069917) because I don't see it supports `<button>` tags inside the translatable messages [here](https://github.com/humhub/translation/blob/master/models/parser/TranslationPurifier.php#L19).
So better is to change the string to:

```php
'status.default' => Yii::t('FcmPushModule.base', 'Default: Push Notifications are not yet enabled.')
    . '<br><button id="enablePushBtn" class="btn btn-primary"><i class="fa fa-unlock"></i>' . Yii::t('FcmPushModule.base', 'Click here to enable') . '</button>',

```

or to:

```php
'status.default' => Yii::t('FcmPushModule.base', 'Default: Push Notifications are not yet enabled.')
    . '<br>'
    . \humhub\widgets\Button::primary(Yii::t('FcmPushModule.base', 'Click here to enable'))
        ->icon('fa-unlock')
        ->id('enablePushBtn')
        ->sm()
        ->loader(false),
```

Reply:
Also  fixed in https://github.com/humhub/fcm-push/pull/44/commits/54bb9cee22c0cbe1e75b5a9f41802103f8b4b5b7